wilsonChan/assignments/fastSlow/lc142/lc142.py

```diff
@@ -1,6 +1,43 @@
 # Problem Statement #
 
-# Given the head of a Singly LinkedList that contains a cycle, write a function to find the starting node of the cycle.
+# Given the head of a Singly LinkedList that contains a cycle, 
```

Comment:
Great overview here and nice job including edge cases and an analysis of the time and space complexity.